src/resolvers/exotics/hosted-git-resolver.js

```diff
@@ -71,6 +71,11 @@ export default class HostedGitResolver extends ExoticResolver {
     throw new Error('Not implemented');
   }
 
+  static getGitSSH(exploded: ExplodedFragment): string {
+    exploded;
+    throw new Error('Not implemented');
```

Comment:
What is actually the purpose of this? Why would you add call to this function and just throw an error in it?